conbench/entities/hardware.py

```diff
@@ -188,9 +188,27 @@ class MachineCreate(marshmallow.Schema):
 
 
 class ClusterCreate(marshmallow.Schema):
-    name = marshmallow.fields.String(required=True)
-    info = marshmallow.fields.Dict(required=True)
-    optional_info = marshmallow.fields.Dict(required=True)
+    name = marshmallow.fields.String(
+        required=True,
+        metadata={
+            "description": "Information related to cluster (e.g. `hosts`, `nodes` or `number of workers`)"
+            "configured to run a set of benchmarks. Used to differentiate between similar"
+            "benchmark runs performed on different sets of hardware"
+        },
+    )
+    info = marshmallow.fields.Dict(
+        required=True,
+        metadata={
+            "description": "`Distinct` name of the cluster, to be displayed on the web UI."
```

Comment:
Just curious: why is the word "Distinct" emphasized/marked as inline code?

Reply:
Based on the description of the issue under #450  

`name: A descriptive name of a cluster. This should be short, but distinctive — it is displayed in the conbench web app UI`

The keyword `distinctive` felt important hence the highlight there. 
(But) Maybe having a check would be better instead (if not already present)